<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sql Manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rsc.qrc">
    <normaloff>:/icons/icons/home.png</normaloff>:/icons/icons/home.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>851</width>
      <height>431</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QGroupBox" name="groupBox_table">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>表格数据</string>
     </property>
     <widget class="QGroupBox" name="groupBox_order">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>241</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>排序</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>238</width>
         <height>26</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_order">
          <property name="text">
           <string>排序字段</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_order"/>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_ascend">
          <property name="text">
           <string>升序</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_descend">
          <property name="text">
           <string>降序</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_filter">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>20</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>数据过滤</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>149</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="radioButton_man">
          <property name="text">
           <string>男</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_girl">
          <property name="text">
           <string>女</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_all_sex">
          <property name="text">
           <string>全显示</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>401</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_record">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>当前记录</string>
     </property>
     <widget class="QGroupBox" name="groupBox_info">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>251</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_empNum">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>54</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>工  号</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_name">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>54</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>姓  名</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_sex">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>54</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>性  别</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_birth">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>54</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>出生日期</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_nation">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>54</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>出生省份</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_dept">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>54</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>部  门</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_salary">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>54</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>工  资</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_empNum">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>10</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="maximum">
        <number>20000</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_name">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>50</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_sex">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>80</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>男</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>女</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="comboBox_birth">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>110</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>1999-5-5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1990-1-1</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="comboBox_nation">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>140</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>上海</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>北京</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>广州</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>成都</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>昆明</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="comboBox_dept">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>170</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>研发部</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>销售部</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>宣发部</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>经营部</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>运维部</string>
        </property>
       </item>
      </widget>
      <widget class="QSpinBox" name="spinBox_salary">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>200</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="maximum">
        <number>99999</number>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_memo">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>381</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>备注</string>
      </property>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>361</width>
         <height>121</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_pic">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>121</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string>照片</string>
      </property>
      <widget class="QLabel" name="label_photo">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>101</width>
         <height>191</height>
        </rect>
       </property>
       <property name="text">
        <string>Pic</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_open_file"/>
   <addaction name="separator"/>
   <addaction name="action_append_line"/>
   <addaction name="action_insert_line"/>
   <addaction name="action_delete_line"/>
   <addaction name="separator"/>
   <addaction name="action_photo_set"/>
   <addaction name="action_photo_clear"/>
   <addaction name="action_salary_increase"/>
   <addaction name="separator"/>
   <addaction name="action_save"/>
   <addaction name="action_undo"/>
   <addaction name="separator"/>
   <addaction name="action_quit"/>
  </widget>
  <action name="action_open_file">
   <property name="icon">
    <iconset resource="rsc.qrc">
     <normaloff>:/icons/icons/folder.png</normaloff>:/icons/icons/folder.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="action_append_line">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="rsc.qrc">
     <normaloff>:/icons/icons/add_circle_outlined.png</normaloff>:/icons/icons/add_circle_outlined.png</iconset>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </action>
  <action name="action_insert_line">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="rsc.qrc">
     <normaloff>:/icons/icons/puzzle.png</normaloff>:/icons/icons/puzzle.png</iconset>
   </property>
   <property name="text">
    <string>插入</string>
   </property>
  </action>
  <action name="action_delete_line">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="rsc.qrc">
     <normaloff>:/icons/icons/delete.png</normaloff>:/icons/icons/delete.png</iconset>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="action_photo_set">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="rsc.qrc">
     <normaloff>:/icons/icons/photo_gallery_outlined.png</normaloff>:/icons/icons/photo_gallery_outlined.png</iconset>
   </property>
   <property name="text">
    <string>设置照片</string>
   </property>
  </action>
  <action name="action_photo_clear">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="rsc.qrc">
     <normaloff>:/icons/icons/photo_album.png</normaloff>:/icons/icons/photo_album.png</iconset>
   </property>
   <property name="text">
    <string>清除照片</string>
   </property>
  </action>
  <action name="action_salary_increase">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="rsc.qrc">
     <normaloff>:/icons/icons/stairs_up.png</normaloff>:/icons/icons/stairs_up.png</iconset>
   </property>
   <property name="text">
    <string>涨工资</string>
   </property>
  </action>
  <action name="action_save">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="rsc.qrc">
     <normaloff>:/icons/icons/save.png</normaloff>:/icons/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="action_undo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="rsc.qrc">
     <normaloff>:/icons/icons/undo.png</normaloff>:/icons/icons/undo.png</iconset>
   </property>
   <property name="text">
    <string>撤销</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="icon">
    <iconset resource="rsc.qrc">
     <normaloff>:/icons/icons/clear.png</normaloff>:/icons/icons/clear.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="rsc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>445</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
